pkg/cmd/server/origin/admission/chain_builder.go

```diff
@@ -198,7 +198,7 @@ func newAdmissionChain(pluginNames []string, admissionConfigFilename string, opt
 		switch pluginName {
 		case lifecycle.PluginName:
 			// We need to include our infrastructure and shared resource namespaces in the immortal namespaces list
-			immortalNamespaces := sets.NewString(metav1.NamespaceDefault)
+			immortalNamespaces := sets.NewString(metav1.NamespaceDefault, metav1.NamespaceSystem)
```

Comment:
This still isn't in sync with upstream.  They have `NamespacePublic` protected too.

Reply:
I don't actually care about public… I care about the namespaces which, when deleted, require cluster-admin intervention

Reply:
I don't really object to public either